Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/FocusHandlerTrigger.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Utility component to hook up Unity events to the OnFocusEnter and OnFocusExit events.
+    /// </summary>
+    public class FocusHandlerTrigger : MonoBehaviour, IMixedRealityFocusHandler
```

Comment:
This seems like something that should be joined with Features/Input/Handlers
And of course renamed to FocusHandler

Reply:
That's a good idea, I'll make that change. I started by using the Interactable behavior (but it makes some assumptions that don't always work and it's pretty heavy for this use) so that's why I put this here.